src/common/opencl.c

```diff
@@ -142,8 +142,12 @@ static inline void _check_clmem_err(const int devid, const cl_int err)
   darktable.opencl->dev[devid].runtime_error |= DT_OPENCL_TUNE_MEMSIZE;
 }
 
-int dt_opencl_get_device_info(dt_opencl_t *cl, cl_device_id device, cl_device_info param_name, void **param_value,
-                              size_t *param_value_size)
+int dt_opencl_get_device_info(
```

Comment:
@jenshannoschwalm @TurboGit I'm really not sure I understand the new coding style that's being introduced.

Reply:
From my understanding of pascals proposed

Function header with one parameter per line if all don't fit into 100 chars.

I think the precise indenting has not been defined.

Static functions with a leading underscore.

Reply:
> I think the precise indenting has not been defined.

From the rest of the code it seems we have the first parameter on the first line, following the function name and opening bracket. Then the following lines align with this first parameter, i.e. the indent is size of function return value + function name + opening bracket. I don't know if this is explicitly documented anywhere, which would be useful if it is going to be enforce/encouraged/rolled out widely. I'm not going to defend this style and don't particularly care tbh, but I don't see much point in updating _just_ the formatting if it doesn't at least create consistency. Especially because I see another update coming in future that "fixes" that again. And pure formatting commits are already annoying whenever you want to do some archeology to find out the original intent of code to see if some cleanup can be done. But this is just me grumping. Sorry! Kindly ignore.

Reply:
If we did one massive automatically generated formatting commit, that doesn't introduce any code changes, we could use `.git-blame-ignore-revs` to not have it mess up blame. This is supposedly supported by https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/ but I can't seem to get it to work in vscode gitlens yet.

And then we'd have to automatically enforce it in later commits.

Reply:
I personally am intersted in "one parameter" per line, that really helps also when writing / reading code.

I also saw that style "next parameter aligned below" , would not be my favourite but.

We should really have a more precise "how to format" policy.

My bad i was mixing code changes with "style"